.github/workflows/ci.yml

```diff
@@ -1112,6 +1137,32 @@ jobs:
           fail_on_unmatched_files: true
           prerelease: ${{ contains(steps.semver.outputs.group1, '-') }}
 
+  release-google-play:
+    name: publish (Google Play)
+    # if: ${{ github.ref == 'refs/heads/main'
+    #      || startsWith(github.ref, 'refs/tags/v') }}
+    needs: ["build"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: build-appbundle-${{ github.run_number }}
+
+      - name: Parse application name from Git repository name
+        id: app
+        uses: actions-ecosystem/action-regex-match@v2
+        with:
+          text: ${{ github.repository }}
+          regex: '^${{ github.repository_owner }}/(.+)$'
+
+      - uses: r0adkll/upload-google-play@v1
+        with:
+          serviceAccountJsonPlainText: ${{ secrets.GOOGLE_PLAY_SERVICE_ACCOUNT }}
+          packageName: ${{ secrets.FCM_BUNDLE_ID }}
+          releaseFiles: ${{ github.workspace }}/${{ steps.app.outputs.group1 }}-android.aab
+          track: internal
+          status: draft
```

Comment:
Discussed: we'll cancel the CI runs on pushes to `main` manually, so that release CI runs build and upload the proper version with `versionCode` properly assigned.